README.md

```diff
@@ -1,9 +1,10 @@
 # Portfolio project
```

Comment:
- Add into your .gitignore file your external files like: .DS_Store, .vscode.

Reply:
I don't understand what did you mean by external files

Reply:
- External files are files that aren't needed into your project and there are persistent like this files hosted in your root directory: .DS_Store, .vscode. For this kind of files or directories are recommended to host in your .gitignore.
![Screen Shot 2021-07-28 at 18 09 18](https://user-images.githubusercontent.com/22258274/127407904-b7b2588b-5653-48ba-a97e-cc60aea24a41.png)

Reply:
Issue not resolved! Kindly ensure to include the directory link of the `DS_Store` in the `.gitignore` file.